Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +204,24 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        empty_tab = Tab.empty
+        tab["poured_from_bottle"] = empty_tab["poured_from_bottle"]
+        tab["HEAD"] = empty_tab["HEAD"]
+        tab["time"] = empty_tab["time"]
+        tab.write
+
+        keg.find {|k| File.utime(File.atime(k), formula_source_time, k) }
```

Comment:
If I'm not mistaken, this changes the modification time of the linked source file of a symbolic link instead of the link itself, thus messes up the `tar` archive meta data. Maybe worth looking into, as symlinks are relatively common?